src/interface/openapi.yaml

```diff
@@ -0,0 +1,809 @@
+openapi: 3.0.0
```

Comment:
Does it make sense to use the api-snippets here?

Reply:
Yes it does. Will do that in this ticket.

Reply:
Cool